packages/dev/serializers/src/glTF/2.0/Extensions/KHR_texture_transform.ts

```diff
@@ -83,14 +83,14 @@ export class KHR_texture_transform implements IGLTFExporterExtensionV2 {
                 return;
             }
 
+            if (babylonTexture.uAng !== 0 || babylonTexture.vAng !== 0) {
+                Tools.Warn(`${context}: Texture ${babylonTexture.name} with rotation in the u or v axis is not supported in glTF.`);
+            }

```

Comment:
```suggestion
            if (babylonTexture.uAng !== 0 || babylonTexture.vAng !== 0) {
                Tools.Warn(`${context}: Texture ${babylonTexture.name} with rotation in the u or v axis is not supported in glTF.`);
                resolve(null);
            }
```

Reply:
In this case the rest of the texture export won't happen, right? Couldn't we still allow the other texture properties to be exported, like scale, etc?

Reply:
The texture would be incorrect. Personally, I would rather the exporter not export the texture than a wrong texture. It's similar to the code right below this.

Reply:
Hmmm that makes sense, then I'll commit your suggestion, thanks! 😄